pkg/rke/k8s_rke_system_images.go

```diff
@@ -9720,6 +9720,48 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/mirrored-pause:3.7",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.22.28",
 		},
+		// Enabled out of band post v2.7.10
```

Comment:
Timing is not 100% clear but most likely this will be released before v2.7.10 so let's update it to v2.7.9 here.
```suggestion
		// Enabled out of band post v2.7.9
```

Reply:
updated